bankgreen/settings.py

```diff
@@ -109,6 +109,13 @@
 
 DATABASES = {"default": {"ENGINE": "django.db.backends.sqlite3", "NAME": BASE_DIR / "db.sqlite3"}}
 
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = "smtp.office365.com"
+EMAIL_USE_TLS = True
+EMAIL_PORT = 587
+EMAIL_HOST_USER = "alerts@bank.green"
+EMAIL_HOST_PASSWORD = "knwfjbzrjfdwmdmj"
```

Comment:
Is this password supposed to be public?

Reply:
Thank you for reviewing it. Good catch. No, I had stored those parameters as env variables. Seems like,it just got misplaced when I did hard reset. Anyways, I have stored them as env variable in .env file now.